serializer_practice/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for serializer_practice project.
```

Comment:
serializer_practice is a bad name.
We should give students an example, how to call their projects and apps